skills/declarative/calendarSkill/calendarskill.dialog

```diff
@@ -716,6 +720,437 @@
         }
       ]
     },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "XdLVa6",
+        "name": "CancelEvent"
+      },
+      "intent": "CancelEvent",
+      "actions": [
+        {
+          "$kind": "Microsoft.SetProperty",
+          "$designer": {
+            "id": "VmYp4l"
+          },
+          "property": "dialog.query",
+          "value": "=turn.activity.text"
+        },
+        {
+          "$kind": "Microsoft.BeginDialog",
+          "$designer": {
+            "id": "jvTDq2"
+          },
+          "activityProcessed": true,
+          "dialog": "AuthenticationDialog"
+        },
+        {
+          "$kind": "Microsoft.BeginDialog",
+          "$designer": {
+            "id": "5aGhIA"
+          },
+          "activityProcessed": true,
+          "dialog": "ResolveDateTimeDialog",
+          "options": {
+            "query": "=dialog.query"
+          },
+          "resultProperty": "dialog.datetime"
+        },
+        {
+          "$kind": "Microsoft.BeginDialog",
+          "$designer": {
+            "id": "mxt6RN"
+          },
+          "activityProcessed": true,
+          "dialog": "SearchEventsDialog",
+          "options": {
+            "start": "=coalesce(dialog.datetime.Start, dialog.datetime.Value)",
+            "end": "=dialog.datetime.End",
+            "title": "=dialog.title",
```

Comment:
This isn't using the title entity from LUIS here so it will always be null. Also, all the LUIS entities need to be stored in the SetProperty step above so that when we call these dialogs through a VA the entities won't be dropped in the AuthenticationDialog.

Reply:
fixed